src/devices/Bno055/Bno055Sensor.cs

```diff
@@ -0,0 +1,486 @@
+            AxisSetting[] axisSettings = new AxisSetting[3];
+            axisSettings[0].Axis = (AxisMap)(retMap & 0x03);
+            axisSettings[1].Axis = (AxisMap)((retMap >> 2) & 0x03);
+            axisSettings[2].Axis = (AxisMap)((retMap >> 4) & 0x03);
+            axisSettings[0].Sign = (AxisSign)(retSign & 0x01);
+            axisSettings[1].Sign = (AxisSign)((retSign >> 1) & 0x01);
+            axisSettings[2].Sign = (AxisSign)((retSign >> 2) & 0x01);
+            SetConfigMode(false);
+            return axisSettings;
+        }
+
+        /// <summary>
+        /// Get the orientation (Eular Angles) X = Heading, Y = Roll, Z = Pitch
+        /// </summary>
+        public Vector3 Orientation
+        {
+            get
+            {
+                var retVect = GetVectorData(Registers.EULER_H_LSB);
+                // If unit is MeterG, then divide by 900, otherwise divide by 16
+                if ((_units & Units.EulerAnglesRadians) == Units.EulerAnglesRadians)
+                    return retVect / 900;
+                else
+                    return retVect / 16;
+            }
+        }
+        /// <summary>
+        /// Get the Magnetometer
+        /// </summary>
+        public Vector3 Magnetometer => GetVectorData(Registers.MAG_DATA_X_LSB) / 16;
+
+        /// <summary>
+        /// Get the gyroscope
+        /// </summary>
+        public Vector3 Gyroscope
+        {
+            get
+            {
+                var retVect = GetVectorData(Registers.GYRO_DATA_X_LSB);
+                if ((_units & Units.AngularRateRotationPerSecond) == Units.AngularRateRotationPerSecond)
+                    return retVect / 900;
+                else
+                    return retVect / 16;
+            }
+        }
+
+        /// <summary>
+        /// Get the accelerometer
+        /// </summary>
+        public Vector3 Accelerometer
+        {
+            get
+            {
+                var retVect = GetVectorData(Registers.ACCEL_DATA_X_LSB);
+                // If unit is MeterG, then no convertion, otherwise divide by 100
+                if ((_units & Units.AccelerationMeterG) == Units.AccelerationMeterG)
+                    return retVect;
+                else
+                    return retVect / 100;
+            }
+        }
+
+        /// <summary>
+        /// Get the linear acceleration
+        /// </summary>
+        public Vector3 LinearAcceleration
+        {
+            get
+            {
+                var retVect = GetVectorData(Registers.LINEAR_ACCEL_DATA_X_LSB);
+                // If unit is MeterG, then no convertion, otherwise divide by 100
+                if ((_units & Units.AccelerationMeterG) == Units.AccelerationMeterG)
+                    return retVect;
+                else
+                    return retVect / 100;
+            }
+        }
+        /// <summary>
+        /// Get the gravity
+        /// </summary>
+        public Vector3 Gravity
```

Comment:
Another acceleration? Now I'm confused...

Reply:
I explained in the documentation. In short: Accelerometer = Gravity + Linear Acceleration.
I've Added more details in the comments.